### DISCUSSION TITLE
Is it possible to use multiple guards (web and admin) and also use Teams?

### DISCUSSION THREAD

**Discussion by geradtroje-env** [spatie/laravel-permission#2681] [Q&A]
I'm trying to create a laravel app where I have two guards (web guard for regular users and an admin guard for admin access to an admin dashboard. On top of that, within the web guard I want to use teams to allow for users to have different roles and permissions per team. 

The problem I'm running into is that on the admin side I don't want to have teams, but if I try to create a role within the admin guard and I don't provide a team_id it fails the role/permissions assignment.

Should I just revert back to a single guard if I'm trying to use teams?

Thanks!

**Comment by parallels999** [comment]
You could leave a default team, or override the migration to allow null in the team_id, and in the login activate/deactivate the team feature by using `config()->set('permission.teams', true/false)`

**Comment by geradtroje-env** [comment]
Decided to revert back to a single guard.
